library/pom.xml

```diff
@@ -378,6 +378,12 @@
         </exclusion>
       </exclusions> 
     </dependency>
+    <dependency>
```

Comment:
Nothing is replaced by the mock server for unit tests. For unit tests we used the actual SQS service and all those jar files are still needed in the regular mode.